Feature/clinical data children page

jhossyd3v opened the pull request:
- Clinical Data Children page
- Fix for internal navigation

Commit:
feat: :construction: Add basic HTML structure for containers

Add containers and sub containers for main structure of page

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/clinical-data-children-page

Commit:
feat: :bento: Add radiographic images

Add radiographic images for accordions on mobile and tabs on desktop

Commit:
fix: :bento: Change names of assets

Change names of assets from hand to wrist  and from ribs to knee

Commit:
feat: :bento: Add charts assets

Add chart assets for mobile and desktop of clinical data children page

Commit:
feat: :lipstick: Add new feature for Highlight Component

Add styles for a required smaller text under content on some contents

Commit:
feat: :construction: Create most of the sections of page

Create HTML structure for most of the sections of clinical data children

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/clinical-data-children-page

Commit:
feat: :bento: Add assets for video position

Add asset of video UI for let the space for video

Commit:
style: :construction: Edit component Highlight Information

Add property for classname and add styles for subcontent

Commit:
feat: :lipstick: Add class for blue color

Add a class in utils for css property color using blue

Commit:
style: :lipstick: Add styles for different screen sizes on children

Add styles for children page, all screen sizes

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/clinical-data-children-page

Commit:
feat: :construction: Add new modal component

Add modal for disability index in clinical data children

Commit:
style: :lipstick: Update styles and HTML structure

Update styles for desktop, add HTML for Modals

Commit:
feat: :bento: Add temp images for clinical data modal

Add temp images for clinical data modal in clinical data children page

Commit:
fix: :bug: Fix internal nav issues

Add a validation if internal nav exists on setNavTopPosition function and add a validation if the href for internal navigation item is #

HackMort reviewed (changes requested):
Issues found:
1. On mobile the spacing is not correct in this section:
![image](https://user-images.githubusercontent.com/6466646/235000828-a8c5c4fe-ee71-498c-8ae6-2ccb94bd07fc.png)
2. Please double check the font-size of these symbols
![image](https://user-images.githubusercontent.com/6466646/235001039-ff699b8f-c297-4775-b602-56410602f614.png)
3. Reduce the padding-top on this section to: 1rem
![image](https://user-images.githubusercontent.com/6466646/235001404-7eabc292-d0dd-4b39-9202-3d242fed6cad.png)
4. The position of the modal trigger is too close to the chart:
![image](https://user-images.githubusercontent.com/6466646/235001510-c39a3d25-672b-44f0-8430-f88028f11717.png)
5. Same comment as above:
![image](https://user-images.githubusercontent.com/6466646/235001629-4f6b3e31-4120-43bd-95b5-5d3c8daefb29.png)
6. The angles and the spacing in this section needs to be adjusted.
![image](https://user-images.githubusercontent.com/6466646/235002116-0a30af6e-0c86-478f-adf1-46c48a2e4501.png)
7. The dot for the list-items should be aligned with the rest of the content:
![image](https://user-images.githubusercontent.com/6466646/235002213-23d8bf07-d3f3-495c-a0ac-5a2bc5a13050.png)
8. Please remove the References Component import and use it with the Layout Prop instead:
![image](https://user-images.githubusercontent.com/6466646/235002347-09dd82ae-5e93-4155-9629-df42a0d1eb01.png)
9. I think we should center this numbers component on 768px until it looks good on desktop:
![image](https://user-images.githubusercontent.com/6466646/235002505-1791b5c6-a6f9-429b-a725-02d5d8f0ff37.png)
10. Have these tabs having the same width on both states, inactive/active so we don't see the "wiggle" effect 
![image](https://user-images.githubusercontent.com/6466646/235002841-f7c471d6-d40a-4d8b-9e11-1045e0a702ba.png)

Almost all the items apply on all screensizes

Commit:
style: :lipstick: Fix spacing in Highlight Information component

Fix spacing between subcontent and footnote for implementations of Highlight Information component in children page

Commit:
style: :lipstick: Fix padding for rgi-scale section

Change padding-top to 1rem such as requested and change padding-right to avoid the text stay outside the shape on some screen sizes

Commit:
style: :lipstick: Fix position of button in Pinch to Zoom

Add enough space between chart and button that triggers modal

Commit:
style: :lipstick: Fix list's dot alignment

Add the correct padding-left for lists in the page

Commit:
style: :construction: Remove extra Reference component

Remove extra reference component, just let the component that comes with Layout

Commit:
feat: :bento: Ad angle separator as an image

Add angle separator as an asset in png format

Commit:
style: :lipstick: Fix Highligh Information

Change styles for highlight information, center from tablet until desktop, change separator for image asset

Commit:
style: :lipstick: Fix width of tab elements

Fix width of tab elements to avoid wiggle effect

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/clinical-data-children-page

HackMort reviewed (changes requested):
2 things:

1. Rever these to be left aligned:
![image](https://user-images.githubusercontent.com/6466646/235204457-8fc054a7-0451-4243-8719-037631f8a2f0.png)
![image](https://user-images.githubusercontent.com/6466646/235204515-343044ca-c0af-49a4-b075-9cd9ff6f273e.png)

2. I see on 1024px everything still has the same layout as the mobile version, I assume this is because we can't fit the desktop version yet, if that's the case, please at least have them in 2 colums per section (similar to desktop). Also change the charts to theri corresponding version of the desktop as well, the vertical ones are taking way too much screen space,

Commit:
style: :lipstick: Fix styles for tablet

Update styles to make two columns on some sections, add styles to images to limit its width

Commit:
style: :lipstick: Fix styles from pinch to zoom component

Remove padding-top since tablet for Pinch to zoom component on children page

Commit:
style: :lipstick: Add styles for section with video for 1024px to 1200px

Add styles to make section with video in 2 columns from 1024px to 1200px

HackMort reviewed (changes requested):
Let's do this:
1. Starting at 1024px, have these sections in 2 columns, 1 column with 2 rows of numbers and the other column with all the "accordion" items. 
![image](https://user-images.githubusercontent.com/6466646/235245845-51ba2dff-6641-42c0-a06e-6b5fa6895e5c.png)
2. Increase space between these sections:
![image](https://user-images.githubusercontent.com/6466646/235246006-ce4706ef-07b6-45b1-9e61-588c111171c6.png)
3. Please try to acomodate the shape to always touch the edge of the screen like in the bigger screens. Also add a bit of more space between the columns
![image](https://user-images.githubusercontent.com/6466646/235246172-8e8bc037-cc83-469e-ae52-ee60c54e9f97.png)
4. Add `display: inline-block;` to `.animated--bg` class, I think that would fix the issue when the text breaks in two lines:
![image](https://user-images.githubusercontent.com/6466646/235246692-f1292462-ba2e-4a5c-8f2d-ac0749997238.png)

Commit:
style: :lipstick: Update spaces between sections

Change columns for first section in page, add more space between columns, fix padding between chart and footnote, fix issue about shape does not touch the edge of the window

HackMort committed:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/clinical-data-children-page

HackMort committed:
Add more space between sections

HackMort reviewed (approved):
I've made some spacing adjustments, now is good to go!
